### ISSUE TITLE
Using YaoBlocks.subroutine instead of deprecated "concentrate" in the tutorials

### ISSUE THREAD

**Issue by xq114** [QuantumBFS/Yao.jl#254]
![phase estimation](https://tutorials.yaoquantum.org/dev/generated/quick-start/2.qft-phase-estimation/)

The quick-start tutorial contains deprecated API. It would be nice to refresh the tutorial. 

![image](https://user-images.githubusercontent.com/42310255/86771329-9cf8f980-c084-11ea-9541-a4f4ce68ddc9.png)

**Comment by Roger-luo** [comment]
good catch! Sorry I forget to update that. Updated in https://github.com/QuantumBFS/tutorials/commit/5f03189ab7b41684ea524ec278729d625531fda5

**Comment by Roger-luo** [comment]
The tutorial CI is broken somehow, I'll close this issue after I finish that so the website gets an update. Thanks for reporting.

XRef: https://github.com/QuantumBFS/tutorials/issues/8
